crates/bevy_ecs/src/event/send_event.rs

```diff
@@ -0,0 +1,15 @@
+use super::{Event, Events};
+use crate::world::{Command, World};
+
+/// A command to Send an arbitrary [`Event`], used by [`Commands::send_event`](crate::system::Commands::send_event).
+pub struct SendEvent<E: Event> {
+    /// The event to send.
+    pub event: E,
+}
```

Comment:
Can we use a simple newtype with `#[derive(Deref, DerefMut)]`?

Reply:
could do, this would go against the style used for all the other command structs though.

Reply:
I'd personally still do it, but it's alright if you disagree.